microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
 
 **China**
 
-*   Prohibited sexual content.
-*   Disputed territory or region references.
-*   Providing or enabling access to content or services that are illegal under applicable local law.
+* Prohibited sexual content.
+* Disputed territory or region references.
+* Providing or enabling access to content or services that are illegal under applicable local law.
+
 
-### 2.12 Age Ratings
+<!-- ------------------------------ -->
+#### 2.12 Age ratings
 
-#### 2.12.1 Mature Content
+<!-- todo: add content between headings -->
+
+
+###### 2.12.1 Mature content
 
 When you submit your extension to [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), you must indicate whether your extension displays content that should be marked "Mature".  When determining the rating for your extension, consider all the content in your app, including user generated content and ads, and to the content that your extension links.  If you indicate that your extension doesn't contain any "Mature" content, you are responsible for maintaining the accuracy of this rating.
 
-Regardless of the rating given to your extension, it must still adhere to all the content requirements of Microsoft Edge Add-ons Developer policies.
+Regardless of the rating given to your extension, it must still adhere to all the content requirements of the Microsoft Edge Add-ons store developer policies.
+
 
-#### 2.12.2 Ratings Change
+###### 2.12.2 Ratings change
 
 If your extension provides content (such as user-generated, retail, or other web-based content) that might be appropriate for a higher age rating than the assigned rating, you must require users to opt into receiving such content by using a content filter or by signing in with a pre-existing account.
 
-### 2.13 Videos
 
-If you submit a promotional video in the listing, it should follow all the content guidelines mentioned in this policy.  If you choose to provide a YouTube link, you must ensure that advertisements are turned off for the specific videos that you want to embed.  For more information on turning on or off ads on YouTube, see [Set your default ad formats](https://support.google.com/youtube/answer/2531367?ref_topic=7072227) and [Ads on embedded videos](https://support.google.com/youtube/answer/132596).
+<!-- ------------------------------ -->
+#### 2.13 Videos
 
+If you submit a promotional video in the listing, it should follow all the content guidelines mentioned in this policy.  If you choose to provide a YouTube link, you must ensure that advertisements are turned off for the specific videos that you want to embed.  For more information on turning on or off ads on YouTube, see [YouTube advertising formats](https://support.google.com/youtube/answer/2531367?ref_topic=7072227) and [Ads on embedded videos](https://support.google.com/youtube/answer/132596).
```

Comment:
"If you submit a promotional video in the listing, then it should strictly adhere to our content guidelines." ``

Reply:
Revising to that, although the revision reads more awkwardly than the existing text:
* "our" content guidelines has objectionable tone, overly personal, and is less specific than "the content guidelines mentioned in this policy".
* "then" is superfluous and doesn't read as well/as tightly.